lib/keycloak-admin/client/user_client.rb

```diff
@@ -125,10 +129,13 @@ def forgot_password(user_id, lifespan=nil)
       execute_actions_email(user_id, ["UPDATE_PASSWORD"], lifespan)
     end
 
-    def execute_actions_email(user_id, actions=[], lifespan=nil)
+    def execute_actions_email(user_id, actions=[], lifespan=nil, redirect_uri=nil, client_id=nil)
+      raise ArgumentError.new("client_id must be defined") if client_id.nil? && !redirect_uri.nil?
       execute_http do
-        lifespan_param = lifespan.nil? ? "" : "lifespan=#{lifespan.seconds}"
-        RestClient.put("#{execute_actions_email_url(user_id)}?#{lifespan_param}", create_payload(actions), headers)
```

Comment:
Adding `redirect_uri` and `client_id` will enable the user to go back to the redirect URL once the user is done with all the actions in the email.